### PR TITLE
Fix kubemark image-registry name assignment

### PR DESCRIPTION
**Pull request by krzysied**
Allows to create kubemark instance without explicitly specifying the project name.
start-kubemark.sh used to not run properly when using project name acquired from the cloud provider.

```release-note
NONE
```

### CODE REVIEW

File: test/kubemark/cloud-provider-config.sh [hunk 1]
```diff
@@ -17,5 +17,5 @@
 CLOUD_PROVIDER="${CLOUD_PROVIDER:-gce}"
 CONTAINER_REGISTRY="${CONTAINER_REGISTRY:-gcr.io}"
 PROJECT="${PROJECT:-}"
-FULL_REGISTRY="${CONTAINER_REGISTRY}/${PROJECT}"
+FULL_REGISTRY="${FULL_REGISTRY:-}"
```
Comment (shyamjvs) [inline test/kubemark/cloud-provider-config.sh]: Could you maybe rename this variable to `KUBEMARK_IMAGE_REGISTRY` to be more explicit?

File: test/kubemark/start-kubemark.sh [hunk 2]
```diff
@@ -207,6 +207,9 @@ function create-and-upload-hollow-node-image {
   CURR_DIR=`pwd`
   cd "${MAKE_DIR}"
   RETRIES=3
+  if [[ -z "${FULL_REGISTRY-}" ]]; then
```
Comment (shyamjvs) [inline test/kubemark/start-kubemark.sh]: I guess you can replace this whole if statement with sth like FULL_REGISTRY="${FULL_REGISTRY:-$CONTAINER_REGISTRY/..}"

**Comment by shyamjvs** [comment]
Also I would suggest having more descriptive PR title. For e.g "Fix kubemark image-registry name assignment"

File: test/kubemark/cloud-provider-config.sh (hunk 1 above)
Comment (krzysied) [reply test/kubemark/cloud-provider-config.sh]: done

File: test/kubemark/start-kubemark.sh (hunk 2 above)
Comment (krzysied) [reply test/kubemark/start-kubemark.sh]: done

**Pull request merged**